### PR TITLE
Turn on instance reaper

### LINKED ISSUE #336: (Regression) Fix AWS Expires/TTL
Expires/TTL system is broken.

- [ ] new pull requests need to tag their instances with an EXPIRES tag and a value
- [ ] the EXPIRES value should initially be a constant set to something like three days?? (get confirmation)

### PR DESCRIPTION
**Pull request by bennett000**
Fixes #336 Connected to #336

### CODE REVIEW

File: src/server/main.js [hunk 1]
```diff
@@ -106,6 +107,9 @@ function createServer(pm, config) {
        */
       bindRoutes(app, pm, dbs);
       clusternatorApi.init(app, dbs.projects);
+      
+      // start reaper @todo wire up the to HUP and/or express restart hooks
+      reaper(pm);
```
Comment (SethDavenport) [inline src/server/main.js]: So currently the reaper will stop if the cluster mayor server restarts?

Comment (SethDavenport) [inline src/server/main.js]: #autocorrectfail

File: src/aws/ec2/ec2-tag.js [hunk 2]
```diff
@@ -125,3 +126,14 @@ function createDeployment(deployment) {
...
 }
+
+/**
+ * @param {string} ttl
+ * @returns {Ec2Tag}
+ */
+function createExpires(ttl) {
+  let safeTTL = parseInt(ttl, 10);
+  // force a value or set to one minute
+  safeTTL = safeTTL >= 0 ? safeTTL : 600000;
```
Comment (SethDavenport) [inline src/aws/ec2/ec2-tag.js]: Is this TTL for closed/merged PRs or does it kick in for open ones too? Just closed/merged is probably ideal.

Comment (bennett000) [inline src/aws/ec2/ec2-tag.js]: It works against any instance with both PR tags and expiration tag so long as the expiration tag is less than the current time stamp

**Comment by bennett000** [comment]
This is working in tests, :shipit:

**Pull request merged**